src/rred_reports/redcap/main.py

```diff
@@ -272,6 +276,8 @@ def _process_calculated_columns(self, entry_year_cols: list[str], export_data: p
         return processed_data
 
     def _add_school_name_column(self, long_df: pd.DataFrame) -> pd.DataFrame:
+        if self._school_aliases:
+            long_df["school_id"] = long_df["school_id"].replace(self._school_aliases, inplace=True)
```

Comment:
If you're using `inplace` then you'll need to do use it without assigning
```suggestion
            long_df["school_id"].replace(self._school_aliases, inplace=True)
```